build.sbt

```diff
@@ -478,24 +479,41 @@ lazy val oteljava = project
   .settings(munitDependencies)
   .settings(scalafixSettings)
 
-lazy val semconv = crossProject(JVMPlatform, JSPlatform, NativePlatform)
-  .crossType(CrossType.Pure)
-  .enablePlugins(BuildInfoPlugin)
-  .in(file("semconv"))
-  .dependsOn(`core-common`)
-  .settings(
-    name := "otel4s-semconv",
-    startYear := Some(2023),
-    // We use opentelemetry-semconv dependency to track releases of the OpenTelemetry semantic convention spec
-    libraryDependencies += "io.opentelemetry.semconv" % "opentelemetry-semconv" % OpenTelemetrySemConvVersion % "compile-internal" intransitive (),
-    buildInfoPackage := "org.typelevel.otel4s.semconv",
-    buildInfoOptions += sbtbuildinfo.BuildInfoOption.PackagePrivate,
-    buildInfoKeys := Seq[BuildInfoKey](
-      "openTelemetrySemanticConventionsVersion" -> OpenTelemetrySemConvVersion
+lazy val `semconv-stable` =
+  crossProject(JVMPlatform, JSPlatform, NativePlatform)
+    .crossType(CrossType.Pure)
+    .enablePlugins(BuildInfoPlugin)
+    .in(file("semconv/stable"))
+    .dependsOn(`core-common`)
+    .settings(
+      name := "otel4s-semconv",
+      startYear := Some(2023),
+      description := "Stable semantic conventions.",
+      // We use opentelemetry-semconv dependency to track releases of the OpenTelemetry semantic convention spec
+      libraryDependencies += "io.opentelemetry.semconv" % "opentelemetry-semconv" % OpenTelemetrySemConvVersion % "compile-internal" intransitive (),
+      buildInfoPackage := "org.typelevel.otel4s.semconv",
+      buildInfoOptions += sbtbuildinfo.BuildInfoOption.PackagePrivate,
+      buildInfoKeys := Seq[BuildInfoKey](
+        "openTelemetrySemanticConventionsVersion" -> OpenTelemetrySemConvVersion
+      )
     )
-  )
-  .settings(munitDependencies)
-  .settings(scalafixSettings)
+    .settings(munitDependencies)
+    .settings(scalafixSettings)
+
+lazy val `semconv-experimental` =
```

Comment:
Nothing depends on the `otel4s-semconv-experimental`. I would expect users to pull this dependency directly into their projects.